apps/boxclk/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+  "id": "boxclk",
+  "name": "Box Clock",
+  "version": "0.01",
+  "description": "A customizable clock with configurable text boxes that can be positioned to show your favorite background",
+  "icon": "app.png",
+  "screenshots": [
+    {"url":"screenshot.png"},
+    {"url":"screenshot-1.png"}
+  ],
+  "type": "clock",
+  "tags": "clock",
+  "supports": ["BANGLEJS2"],
+  "readme": "README.md",
+  "allow_emulator": true,
+  "storage": [
+    {"name":"boxclk.app.js","url":"app.js"},
+    {"name":"boxclk.json","url":"boxclk.json"},
```

Comment:
you probably want to move this into:

```
"data" : [
{"name":"boxclk.json","url":"boxclk.json"},
]
```

Because then upgrades won't overwrite any settings stored in this file?